Prose/ProseLib/Sources/AuthenticationFeature/AuthenticationScreen.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  AuthenticationView.swift
+//  Prose
+//
+//  Created by Marc Bauer on 01/04/2022.
+//
+
+import ComposableArchitecture
+import SwiftUI
+
+public struct AuthenticationScreen: View {
```

Comment:
Well, it's not an `App`, it's not a `Scene` and it's a `View`, so it could be called `XxxView`. However, I like to distinguish views (understand small pieces of UI) from complete screens. I think it helps understanding what the view is in the big picture.

And Apple does not name every view with the `View` suffix, so I don't find it shocking. But that's up to you 🤷🏻‍♂️

Reply:
Actually I'm realising my terminology comes from iOS "screens", so I get it's a bit ambiguous on macOS.